src/components/CreateArticleForm.jsx

```diff
@@ -34,9 +34,11 @@ const CreateArticleForm = ({ onCreateMessage }) => {
     event.preventDefault();
     const response = await Articles.create(article);
     onCreateMessage(response);
-    setArticle(InitialArticle);
-    setFileName("");
-    setTimeout(() => onCreateMessage(""), 4000);
+    if (response === "Article succesfully created!") {
+      setArticle(InitialArticle);
+      setFileName("");
+      setTimeout(() => onCreateMessage(""), 4000);
```

Comment:
We can clean up the message in any case, even if it's an error, otherwise it will not go away
In short, you may move this line out of the conditional